flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/descriptors/RowtimeTest.scala

```diff
@@ -87,7 +87,7 @@ class RowtimeTest extends DescriptorTestBase {
         "B0b3JzLlJvd3RpbWVUZXN0JEN1c3RvbUFzc2lnbmVyeDcuDvfbu0kCAAB4cgBHb3JnLmFwYWNoZS5mbGluay" +
         "50YWJsZS5zb3VyY2VzLndtc3RyYXRlZ2llcy5QdW5jdHVhdGVkV2F0ZXJtYXJrQXNzaWduZXKBUc57oaWu9A" +
         "IAAHhyAD1vcmcuYXBhY2hlLmZsaW5rLnRhYmxlLnNvdXJjZXMud21zdHJhdGVnaWVzLldhdGVybWFya1N0cm" +
-        "F0ZWd5mB_uSxDZ8-MCAAB4cA")
```

Comment:
What is the reason for this mismatch?

Reply:
Previously, the `WatermarkStrategy` extends a Scala `Serializable`. But now it extends the Java Serializable. Also, we make it extends `Descriptor`. The class has been changed.
Same reason for `TimestampExtractor`.